pkpdapp/pkpdapp/models/biomarker_type.py

```diff
@@ -0,0 +1,28 @@
+#
+# This file is part of PKPDApp (https://github.com/pkpdapp-team/pkpdapp) which
+# is released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+
+from django.db import models
+
+
+class BiomarkerType(models.Model):
+    """
+    A biomarker type, for example "concentration in mg". Each
+    :model:`pkpdapp.Biomarker` is assigned on of these types
+    """
+
+    UNIT_CHOICES = [
+        ('mg', 'type1'),
+    ]
+    name = models.CharField(
+        max_length=100, help_text='name of the biomarker type'
+    )
+    unit = models.CharField(
+        max_length=2, choices=UNIT_CHOICES,
```

Comment:
Is two characters sufficient for all units? Worth increasing to (say) 4?

Reply:
yup, increased this to 5